src/metabase/native_query_analyzer.clj

```diff
@@ -110,6 +135,42 @@
                                                     [:= :f.active true]
                                                     (into [:or] (map table-query tables))]}))))
 
+;; NOTE: In future we would want to replace [[considered-drivers]] and [[macaw-options]] with (a) driver method(s),
+;; but given that the interface with Macaw is still in a state of flux, and how simple the current configuration is,
+;; we defer extending the public interface and define the logic locally instead. Macaw itself is not battle tested
+;; outside this same narrow range of databases in any case.
+
+(def ^:private considered-drivers
+  "Since we are unable to ask basic questions of the driver hierarchy outside of that module, we need to repeat"
```

Comment:
```suggestion
  "Since we are unable to ask basic questions of the driver hierarchy outside of that module, we need to explicitly mention sub-types"
```